examples/jsm/renderers/webgpu/WebGPURenderPipelines.js

```diff
@@ -100,7 +100,7 @@ class WebGPURenderPipelines {
 			const disposeCallback = onMaterialDispose.bind( this );
 			materialProperties.disposeCallback = disposeCallback;
 
-			material.addEventListener( 'dispose', onMaterialDispose.bind( this ) );
+			material.addEventListener( 'dispose', disposeCallback );
```

Comment:
`onMaterialDispose()` uses the `this` reference. I don't think your change works if you are not using `bind()`.

Reply:
The `bind()` call was doubled. `disposeCallback ` already use `bind()`